include/networkit/algebraic/MatrixTools.hpp

```diff
@@ -129,6 +133,22 @@ NetworKit::Graph matrixToGraph(const Matrix &matrix) {
     return G;
 }
 
+/**
+ * Converts the @a other matrix into a equivalent matrix of the template parametrized type.
+ * @param other The input Matrix.
+ * @return Matrix of type DenseMatrix, CSRMatrix or DynamicMatrix.
+ */
+template <class ToMatrix, class FromMatrix>
+ToMatrix matrixTo(const FromMatrix &other) {
+    int nRows = other.numberOfRows();
+    int nCols = other.numberOfColumns();
+    ToMatrix M(nRows, nCols, 0.0);
```

Comment:
the default/zero value of `M` should be the default/zero value of `other` ?
Also, avoid single letter variables